.github/workflows/pages-build-and-deploy.yml

```diff
@@ -0,0 +1,43 @@
+name: pages-build-and-deploy
```

Comment:
Very nice. Where are the doc files pushed? Did you have a chance to see the results in your fork?

Reply:
I'm actually not entirely sure, I think in the `github-pages` repository environment. See https://michael-the1.github.io/diepvries/